commit 302e8210b865576b35164f48bd57e003f6674dd1
Author: FalsinSoft <falsinsoft@gmail.com>
Date:   Wed Jan 11 20:58:04 2023 +0100

    Fixed 'invalid_argument' is not a member of 'std' error on Visual Studio

diff --git a/core/src/Matrix.h b/core/src/Matrix.h
index 8b04bb1a..0c7c4e88 100644
--- a/core/src/Matrix.h
+++ b/core/src/Matrix.h
@@ -9,6 +9,7 @@
 #include "Point.h"
 #include "ZXAlgorithms.h"
 
+#include <stdexcept>
 #include <algorithm>
 #include <cassert>
 #include <vector>
